src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -169,6 +170,14 @@ void scrollBreadth(double deltaBreadth) {
         setBreadthOffset(breadthOffset.get() + deltaBreadth);
     }
 
+    void scrollLengthToPixel(double pixel) {
+        setLengthOffset(pixel);
+    }
+
+    void scrollBreadthToPixel(double pixel) {
+        setBreadthOffset(pixel);
+    }
```

Comment:
The above two methods seem unnecessary&mdash;they are just aliases for other methods.